modules/account-lib/src/coin/dot/transactionBuilder.ts

```diff
@@ -28,6 +35,7 @@ export abstract class TransactionBuilder extends BaseTransactionBuilder {
   protected _registry: TypeRegistry;
   protected _method?: TxMethod;
   protected __material?: Material;
+  protected _signatures: Interface.Signature[] = []; // only support single sig for now
```

Comment:
We might want to distinguish this `signatures` from the one we had in `transaction.ts`, one is for builder's functionality and the other is something else

Reply:
added a comment. lmk if you think that's enough

Reply:
yeah it looks good!